tracer/test/snapshots/AspNetMvc5TestsModuleOnly.Classic.__path=_statuscode_503_statusCode=503.verified.txt

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet.request,
+    Resource: GET /statuscode/?,
```

Comment:
FYI for other reviewers, this is one example where we get different resource names when we don't run MVC:

- With MVC: `GET /statuscode/{value}`
- Without MVC: `GET /statuscode/?`